src/adapter/src/catalog/builtin.rs

```diff
@@ -1567,13 +1567,17 @@ pub static MZ_CLUSTER_REPLICA_METRICS: Lazy<BuiltinTable> = Lazy::new(|| Builtin
         .with_column("memory_bytes", ScalarType::UInt64.nullable(true)),
 });
 
-pub static MZ_STORAGE_HOST_METRICS: Lazy<BuiltinTable> = Lazy::new(|| BuiltinTable {
-    name: "mz_storage_host_metrics",
+pub static MZ_STORAGE_REPLICA_METRICS: Lazy<BuiltinSource> = Lazy::new(|| BuiltinSource {
```

Comment:
+ I think "Replica" is more meaningful to users and makes it easier to connect Storage resources and Compute resources as like concepts.